README.md

```diff
@@ -257,6 +257,13 @@ filters:
 The pipeline means it will forward traffic to 3 backend endpoints, using the
 `roundRobin` load balance policy.
 
+You can also create them using `egctl create httpproxy` command.
+```bash
+egctl create httpproxy demo --port 10080 \ 
+  --rule="/pipeline=http://127.0.0.1:9095,http://127.0.0.1:9096,http://127.0.0.1:9097"
+```
+this command will create `HTTPServer` `demo-server` and `Pipeline` `demo-pipeline-0` which work exactly same to `server-demo` and `pipeline-demo`. See more about [`egctl create httpproxy`](./doc/egctl-cheat-sheet.md#create-httpproxy).
+
```

Comment:
propose not adding `-server` suffix to the name, user may get confused what he/she has created.
for the pipeline names, also propose not using the `pipeline-x` suffixes.

Reply:
good idea. I will update that!